opentera-webrtc-native-client/OpenteraWebrtcNativeClient/python/package/CMakeLists.txt

```diff
@@ -1,41 +1,118 @@
-# Generate files
-configure_file(setup.py.in setup.py)
-configure_file(README.md.in README.md) 
-configure_file(__init__.py.in __init__.py) 
+set(PYTHON_LIB_NAME opentera_webrtc_native_client)
 
-set (PYTHON_LIB_NAME opentera_webrtc_native_client)
+set(INSTALL_ON_BUILD ${OPENTERA_WEBRTC_NATIVE_CLIENT_PYTHON_INSTALL_ON_BUILD})
+if(INSTALL_ON_BUILD)
+    set(ALL_OPTION ALL)
+else()
+    set(ALL_OPTION)
+endif()
+
+set(LICENSE_FILE ${CMAKE_CURRENT_SOURCE_DIR}/../../../../LICENSE)
+
+# Configuration of cmake template files
+configure_file(setup.py.in ${CMAKE_CURRENT_BINARY_DIR}/setup.py.in2)
+# configure_file can't use generator expressions
+file(GENERATE
+    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/opentera_webrtc_native_client/setup.py
+    INPUT ${CMAKE_CURRENT_BINARY_DIR}/setup.py.in2
+)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/README.md.in ${CMAKE_CURRENT_BINARY_DIR}/opentera_webrtc_native_client/README.md)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/__init__.py.in ${CMAKE_CURRENT_BINARY_DIR}/opentera_webrtc_native_client/__init__.py)
+configure_file(${LICENSE_FILE} ${CMAKE_CURRENT_BINARY_DIR}/opentera_webrtc_native_client/LICENSE)
+
+# # Signaling server files
+# set(signaling_server_source_files
+#     ../../../../signaling-server/signaling_server.py
+#     ../../../../signaling-server/room_manager.py
+#     ../../../../signaling-server/requirements.txt
+#     ../../../../signaling-server/README.md
+# )
+
+# install(FILES ${signaling_server_source_files} DESTINATION python/signaling-server)
```

Comment:
I prefer the sub package option. @mamaheux ?

Reply:
I would not put the signaling-server in any CMake file, since it's a Python web server. @doumdi why there is an install here?

Reply:
@mamaheux I think it can be removed. Never used it anyway.

Reply:
> I would not put the signaling-server in any CMake file, since it's a Python web server. @doumdi why there is an install here?

The CMakeLists could generate the wheel installer (and maybe install it, like its done for the native client? It would be easier than having to navigate to the subfolder and then invoke `pip install .`